### ISSUE TITLE
Jetpack Search prevents URL filtering 

### ISSUE THREAD

**Issue by jessefriedman** [Automattic/jetpack#17704]
1. Go to a WordPress site with Jetpack Search enabled and type into the URL /author/authorname?s=searchterm
2. Jetpack Search will take over the search because of the ?s= param but will not honor the URL filter of author name

#### What I expected
Normally, WordPress Core allows you to search within authors, tags, categories, or on WordPress.com mentions.  This allows you to quickly find something buy typing jesse.blog/category/articles/?s=Jetpack.  This URL would automatically allow me to see results for the query "Jetpack" for posts that are categorized in "articles". 

#### What happened instead
Jetpack Search grabs the search param and just shows me the results for the entire site without filtering down into the URL filtered param.

**Screenshots**
<img width="1125" alt="Screen Shot 2020-11-03 at 3 55 22 PM" src="https://user-images.githubusercontent.com/1749986/98039645-01d31f00-1ded-11eb-8370-378a57d6654f.png">
These results are not authored by the user "jesserfriedman"

**Comment by gibrown** [comment]
@jessefriedman do you think we should also open the Jetpack Search modal when there is no search param. e.g. example.com/author/gibrown ? Or make that an option? Takes the control away from the theme, but gives the user a faster discovery/exploration interface.

Related question, does your opinion change once we have Gutenberg customization of the search overlay?

**Comment by jessefriedman** [comment]
@gibrown My preference would be to have the modal open up and for Jetpack Search to absorb the URL param filtering.  In other words, if I do https://jetpack.com/author/jesserfriedman?s=publicize I'd like to see the modal open and the filter for "author" prechecked for "jesserfriedman" and the results for publicize showing.  

This will allow me to get to my results as soon as possible. 

My opinion does not change for the Gutenberg customizations.  I don't think we should ever bifurcate the search experience. One search results UI for all searches whether it's through queries in a search box or "URL filtering".

**Comment by lezama** [comment]
Reported here too: https://a8c.slack.com/archives/C010KDAPG49/p1614975422265900

**Comment by erikdemarco** [comment]
> @gibrown My preference would be to have the modal open up and for Jetpack Search to absorb the URL param filtering. In other words, if I do https://jetpack.com/author/jesserfriedman?s=publicize I'd like to see the modal open and the filter for "author" prechecked for "jesserfriedman" and the results for publicize showing.
> 
> This will allow me to get to my results as soon as possible.
> 
> My opinion does not change for the Gutenberg customizations. I don't think we should ever bifurcate the search experience. One search results UI for all searches whether it's through queries in a search box or "URL filtering".

I don't agree with this. This will contradict what "jetpack search" do. It only needs to handle when search params exist. From UX perspective it also very bad. Like when people clicking a link they expect a content to show up. At split second people will be shown "no result" before jetpack modal shows up.

From search engine is also maybe bad. It shows a modal popup. Which against seo. And the content is similar to disqus widget (loading the content via js) which make google hard to crawl. We should make it easy for search engine to crawl the content

My preference is. When there is no search param, let jetpack search handle it (but with instant search disabled). When search param exist, let jetpack handle it (but with instant search enabled)

**Comment by iandunn** [comment]
The also affects date-based queries like `/2020/?s=foo` and `/2020/03/15/?s=bar`

**Comment by geektzu** [comment]
Hi team,
There is a merchant reporting this bug that is relevant to this issue. On the latest arrival page, the sort drop-down menu used to fire query string like "?orderby=popularity&paged=1&post_type=product&s="

https://diabolikdvd.com/product-category/latestarrivals/
